src/Movie.php

```diff
@@ -0,0 +1,29 @@
+<?php namespace Acme;
+
+use Psr\Http\Message\ResponseInterface;
+
+class Movie {
+    public string $Title, $Year, $Response;
```

Comment:
The Movie Class should not have the `$Response` itself here. 

Think about this: What if next week there is a requirement to fetch movies from DB instead from API Call? is the $response related with a Movie?